project_management/Technical_setup_when_starting_a_new_project.md

```diff
@@ -70,6 +70,16 @@ Setup automerge github action
   - create github automerge action
     - copy auto-merge.yml, dependabot.yml, workflows/auto-merge-dependabot.yml from [muffi/.github](https://github.com/abtion/muffi/tree/main/.github) into [clientname]-[projectname]/.github
 
+Repo settings
+  - Disallow `Allow rebase merging` (because it is incompatible with `Require signed commits`)
+  - Enable `Allow auto-merge`
+  - Enable `Automatically delete head branches Loading`
+  - Enable `Always suggest updating pull request branches`
+  - Setup `Rules`. Add a branch ruleset
```

Comment:
This could be implemented in the whole organization

Reply:
https://github.com/organizations/abtion/settings/rules
> Organization rulesets won't be enforced [until you upgrade this organization account to GitHub Enterprise](https://github.com/organizations/abtion/billing/plans)

But maybe then there's a default rule that these instructions can just tell you to enable?